dactyl-config.yml

```diff
@@ -445,6 +443,22 @@ pages:
         targets:
           - local
 
+    -   md: concepts/consensus-network/consensus-protections.md
+        html: consensus-protections.html
+        funnel: Docs
+        doc_type: Concepts
+        category: Consensus Network
+        blurb: Learn how the XRP Ledger Consensus Protocol is protected against various problems and attacks that may occur in a decentralized financial system.
```

Comment:
For some reason this page isn't building - perhaps it needs the `targets: - local` config? I've been away from the dev portal for so long, I can't remember. 😢

Reply:
Good catch.